crates/core/src/dht/dht_reducers.rs

```diff
@@ -251,9 +246,12 @@ pub mod tests {
     fn can_add_links() {
         let context = test_context("bob", None);
         let store = test_store(context.clone());
-        let entry = test_entry();
+        let entry_wh = test_entry_with_header();
+        let entry = entry_wh.entry.clone();
+
+        let store = reduce_hold_entry(&store.dht(), &ActionWrapper::new(Action::Hold(entry_wh)))
```

Comment:
Had to rewrite this test to use proper mutations of the DHT store. This is a better way of doing things anyway.